src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
 }
+
+.header {
+  justify-content: space-between;
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+  height: 40px;
+  width: 40px;
+}
+
+.nav__item {
+  margin-left: 20px;
+  position: relative;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
```

Comment:
position is not needed here and its better to write `.nav__item:not(:last-child) { margin-left: 20px ;}` ,  check here https://developer.mozilla.org/en-US/docs/Web/CSS/:not